runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -198,6 +205,12 @@ public void onConfigChanged(ConfigChanged event)
 		});
 	}
 
+	@Subscribe
+	public void onPlayerSpawned(PlayerSpawned event)
```

Comment:
Are you sure this doesn't also fire when other players "spawn"?

Reply:
I suspected the same thing, so I waited inside the GE to find out, and none of the players teleporting in were triggering this callback.

Reply:
Scratch that -- it actually *does* fire for every player nearby, I wasn't checking it properly.

Reply:
Sweet. I also like the config for yours. I'll close mine out. Do we know what format we want to display the username? I saw you went with "RuneLite - name" and I went with "RuneLite (name)"

Reply:
I picked a hyphen format because it works on both sides, since it looks a bit odd to have the window name start with a parenthesis. Let me take a minute to add a similar justLogged check to the one you implemented, we don't want this callback shooting off constantly for no benefit 👅